.travis.yml

```diff
@@ -111,7 +111,8 @@ before_script:
     - if type ccache; then ccache -s; fi
 script:
     - cd $TRAVIS_BUILD_DIR/build && ../scripts/tests.sh $TRAVIS_TESTS
-
+after_success:
+    - cd $TRAVIS_BUILD_DIR/build && bash <(curl -s https://codecov.io/bash) -a '>/dev/null 2>&1'
```

Comment:
This basically grants the people at codecov access to our deployment keys. Is that bad?